ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-main.yaml

```diff
@@ -1,6 +1,6 @@
 base_images:
   base:
-    name: ubi
+    name: ubi-minimal
```

Comment:
@sudomakeinstall2 you need to make this change in other branch config files too..
https://github.com/openshift/release/tree/master/ci-operator/config/openshift-kni/cluster-group-upgrades-operator

Reply:
fixed